docker/cygnus-common/Dockerfile

```
@@ -36,7 +36,7 @@ ENV CYGNUS_API_PORT "8081"
 # PR https://github.com/telefonicaid/fiware-cygnus/pull/1801 should be also done in this Dockerfile
 # NOTE: Configure correctly GIT_URL_CYGNUS and GIT_REV_CYGNUS for each git branch/fork used
 ENV GIT_URL_CYGNUS "https://github.com/telefonicaid/fiware-cygnus.git"
-ENV GIT_REV_CYGNUS "master"
+ENV GIT_REV_CYGNUS "task/purge_error_table"
```

```suggestion
ENV GIT_REV_CYGNUS "master"
```

(Seems to be a local change not changed before pushing the commit to github)